asm/f32/benchDot_test.go

```diff
@@ -16,7 +16,7 @@ func BenchmarkDotUnitary(t *testing.B) {
 	for _, v := range []int64{1, 2, 3, 4, 5, 10, 100, 1e3, 5e3, 1e4, 5e4} {
 		t.Run(fmt.Sprintf("%s-%d", name, v), func(b *testing.B) {
 			x, y := x[:v], y[:v]
-			b.SetBytes(128 * v)
+			b.SetBytes(32 * v)
```

Comment:
Sorry for asking again, where does the 32 comes from? Or how do you interpret "the number of bytes processed in a single operation" in case of `Dot`?